tests/security/linux_security_sensor/velociraptor_client_event_collect.pm

```diff
@@ -87,39 +87,107 @@ sub run {
     # add client monitoring
     my @artifacts = qw(SUSE.Linux.Events.DNS SUSE.Linux.Events.ExecutableFiles SUSE.Linux.Events.ImmutableFile SUSE.Linux.Events.NewFiles SUSE.Linux.Events.NewFilesNoOwner SUSE.Linux.Events.NewHiddenFile SUSE.Linux.Events.NewZeroSizeLogFile SUSE.Linux.Events.Packages SUSE.Linux.Events.ProcessStatuses SUSE.Linux.Events.SSHLogin SUSE.Linux.Events.Services SUSE.Linux.Events.SshAuthorizedKeys SUSE.Linux.Events.SystemLogins SUSE.Linux.Events.TCPConnections SUSE.Linux.Events.Timers SUSE.Linux.Events.UserAccount SUSE.Linux.Events.UserGroupMembershipUpdates);
     foreach my $artifact (@artifacts) {
-        assert_script_run("velociraptor-client --api_config ~/api.config.yaml query 'SELECT add_client_monitoring(artifact=\"$artifact\") FROM scope()'");
-    }
+        if ($artifact == 'SUSE.Linux.Events.Packages' || $artifact == 'SUSE.Linux.Events.SshAuthorizedKeys') {
+            script_run("velociraptor-client --api_config ~/api.config.yaml query 'SELECT add_client_monitoring(artifact=\"$artifact\", parameters=dict(period=\"10\")) FROM scope()' > /dev/null");
 
-    # generetae client events
-    sleep 10;
-    my $i = 0;
-    while ($i < 10) {
-        assert_script_run "echo 'Client Event exec' >> /home/genfile$i.sh";
-        assert_script_run "chmod +x /home/genfile$i.sh";
-        assert_script_run "echo 'Client Event exec' >> /tmp/genfile$i.sh";
-        assert_script_run "chmod +x /tmp/genfile$i.sh";
-        $i++;
+        }
+        else {
+            script_run("velociraptor-client --api_config ~/api.config.yaml query 'SELECT add_client_monitoring(artifact=\"$artifact\") FROM scope()' > /dev/null");
+        }
     }
-    sleep 60;
+
+    systemctl "restart velociraptor-client";
+    systemctl "status velociraptor-client";
```

Comment:
the client probably needs a bit of time to start before we start triggering - 10 sec should be enough. The start on L73 can be removed too.